flight/PiOS/Common/pios_ms5611.c

```diff
@@ -434,7 +433,7 @@ static void PIOS_MS5611_Task(void *parameters)
 		PIOS_MS5611_ClaimDevice();
 		PIOS_MS5611_StartADC(PRESSURE_CONV);
 		PIOS_Thread_Sleep(PIOS_MS5611_GetDelay());
-		read_adc_result = PIOS_MS5611_ReadADC();
```

Comment:
Don't need this at all anymore if you do this? You removed all the places it could be written non-zero and it just does:
```c
 		if (read_adc_result == 0) {
 			PIOS_Queue_Send(dev->queue, &data, 0);
 		}
```

We will now send through stale values as if they were fresh ones when the read fails.